### PR TITLE
[UX] Move double tap to gesture manager

### PR DESCRIPTION
**Pull request by Frenzie**
Also replaces the default with +10/-10 pages instead of prev/next chapter.

Closes <https://github.com/koreader/koreader/issues/2721>.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6322)

### CODE REVIEW

File: frontend/apps/reader/modules/readergesture.lua [hunk 1]
```diff
@@ -352,6 +360,41 @@ function ReaderGesture:addToMainMenu(menu_items)
...
+                enabled_func = function() return self.ges_mode == "gesture_reader" end,
+                sub_item_table = self:buildMenu("double_tap_left_side", self.default_gesture["double_tap_left_side"]),
+            },
+            {
+                text_func = function() return actionTextFunc("double_tap_right_side", _("Right side")) end,
+                sub_item_table = self:buildMenu("double_tap_right_side", self.default_gesture["double_tap_right_side"]),
+            },
+            {
+                text_func = function() return actionTextFunc("double_tap_top_left_corner", _("Top left")) end,
+                enabled_func = function() return self.ges_mode == "gesture_reader" end,
```
Comment (poire-z) [inline frontend/apps/reader/modules/readergesture.lua:380]: Why does that top left one has this enabled_func, and not the others?

Comment (Frenzie) [reply frontend/apps/reader/modules/readergesture.lua:380]: Copy paste oversight. Top left (tap) isn't implemented in the gesture manager in the file manager.

File: frontend/apps/reader/modules/readergesture.lua [hunk 2]
```diff
@@ -191,6 +191,12 @@ function ReaderGesture:init()
         one_finger_swipe_top_edge_left = "ignore",
         one_finger_swipe_bottom_edge_right = "ignore",
         one_finger_swipe_bottom_edge_left = "ignore",
+        double_tap_left_side = self.ges_mode == "gesture_reader" and "prev_chapter",
+        double_tap_right_side = self.ges_mode == "gesture_reader" and "next_chapter",
```
Comment (poire-z) [inline frontend/apps/reader/modules/readergesture.lua:195]: > Also replaces the default with +10/-10 pages instead of prev/next chapter.

Not yet done ?

Comment (Frenzie) [reply frontend/apps/reader/modules/readergesture.lua:195]: Whoops, I thought I'd picked one. I couldn't quite decide whether it was better to stick to the current or change it. Thanks for being vigilant. :-)

(Any thoughts?)

Comment (poire-z) [reply frontend/apps/reader/modules/readergesture.lua:195]: (not really, I have double tap disabled - May be +10/-10 is better for skimming, some books might not have any chapter)

Comment (Frenzie) [reply frontend/apps/reader/modules/readergesture.lua:195]: Yeah, I don't use it either. As I said in the referenced issue I don't like the delay, even when I make it about a 100. Much lower than that isn't practical.

**Comment by Frenzie** [comment]
@poire-z Any other comments?

File: frontend/apps/reader/modules/readergesture.lua [hunk 3]
```diff
@@ -243,6 +249,8 @@ function ReaderGesture:init()
     local mirrored_if_rtl = {
         tap_top_left_corner = "tap_top_right_corner",
         tap_right_bottom_corner = "tap_left_bottom_corner",
+        double_tap_left_side = "double_tap_right_side",
+        double_tap_right_side = "double_tap_left_side",
```
Comment (poire-z) [inline frontend/apps/reader/modules/readergesture.lua:253]: (I don't remember why I just set tap_top_left_corner  and tap_right_bottom_corner, and not their counterparts.
Oh, but I switch them, so one of the swapped pair is enough. But it's strange I made that kinda crossed, and not both: left_corner = "right" ...)

**Pull request merged**